.project-keeper.yml

```diff
@@ -20,9 +20,11 @@ sources:
       artifactId: "extension-manager-parent-pom"
       version: "${revision}"
       relativePath: "../pom.xml"
-# Activate registry when https://github.com/exasol/project-keeper/issues/398 is fixed
+# Activate NPM modules when https://github.com/exasol/project-keeper/issues/398 is fixed
 #  - type: npm
```

Comment:
Ticket exasol/project-keeper#398 is closed. You can reactivate this code.

Reply:
✅